mlflow_faculty/utils.py

```diff
@@ -0,0 +1,11 @@
+from mlflow.entities import Experiment, LifecycleStage
```

Comment:
maybe rename this as `mlflow_converters`?

Reply:
Good call